### DISCUSSION TITLE
is_leaf why not support DataType::Decimal256

### DISCUSSION THREAD

**Discussion by jun0315** [apache/arrow-rs#4262] [Q&A] [answered]
https://github.com/apache/arrow-rs/blob/master/parquet/src/arrow/arrow_writer/levels.rs#LL90C15-L90C41

Why is_leaf is not support DataType::Decimal256?

**Comment by tustvold** [comment]
Decimal256 was a relatively recent addition to the arrow specification, and so there are various code paths that haven't yet been updated to make use of it. There is no particular reason why it cannot be supported

**Reply by tustvold** [reply, marked as answer]
FWIW Decimal256 is supported now
